rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/RholangMethodsCostsSpec.scala

```diff
@@ -765,13 +766,215 @@ class RholangMethodsCostsSpec
     }
   }
 
+  "reducer" when {
+    "it works with BigInt" should {
+      val table = Table(
+        ("left", "right"),
+        (gBigInt("225"), gBigInt("25")),
+        (
+          gBigInt("9999999999999999999999999999999999999999"),
+          gBigInt("-9999999999999999999999999999999999999999")
+        ),
+        (
+          gBigInt("0"),
+          gBigInt("9999999999999999999999999999999999999999")
+        ),
+        (
+          gBigInt("123"),
+          gBigInt("-9999999999999999999999999999999999999999")
+        )
+      )
+
+      "charge expression `-x` as `size(x)`" in {
```

Comment:
I confused about this tests. It checks cost accounting for BigInt manipulation? Why cost of `-x` is `size(x)`? Should we consider negation? Is there difference in cost for `-x` and `x`?

Reply:
You need to create a new BigInt value for creating `-x` from `x`. You need to copy each byte of `x` for this. size(x) is size of bytes which need's for store `x` value.